content/docs/ops/runbook/troubleshooting-bosh.md

```diff
@@ -33,15 +43,13 @@ documentation](https://github.com/concourse/fly#installing-from-the-concourse-ui
 1. Click the plus button to start your own build of your selected job. Remember
    the build number as you'll be referencing it in the `builds` command.
 
-If you haven't already, set a target to your concourse using the following command
+If using the command-line is easier, you can trigger a job with the following
+command instead of going through the UI.
 
-```sh
-$ fly --target <YOUR_CONCOURSE_TARGET_NAME> login --concourse-url <CONCOURSE_URL> (e.g. https://ci.example.com)
-```
-You should now see that target when you issue the following command.
-```sh
-$ fly targets
+```shell
+fly -t <YOUR_CONCOURSE_TARGET_NAME> trigger-job -j jumpbox/<JOB_NAME>
```

Comment:
Should we simply this as well?  Just state that when you run this command, it prints your build number out:

`started jumpbox/<JOB_NAME> #<BUILD_NUMBER>`

and you can skip all the complexity below by then running

`fly -t <YOUR_CONCOURSE_TARGET_NAME> intercept -j jumpbox/<JOB_NAME> -b <BUILD_NUMBER>` 

and selecting the `task` container